Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift

```diff
@@ -200,7 +200,9 @@ class BalancesViewController: LoadableViewController, UITableViewDelegate, UITab
             updateSection(indexPath.section)
         }
         cell.onImport = { [unowned self] in
-            let vc = CreatePasscodeViewController { [weak self] in
+            AppSettings.passcodeBannerDismissed = true
```

Comment:
I think this is wrong, as per spec the banner should be dismissed when user imports the key

Reply:
Sorry if the spec confusing in this place. Here how it is stated:
```
User is reminded to set up a passcode with a banner in coins assets screen if all the following cases apply:
user has imported signing key
passcode was not set
banner was not closed / skipped
```
Starting setting up and not finishing it is "skipped" here. So the logic is the same as with "import signing key" banner.

Reply:
I see.